modules/nw-ingress-reencrypt-route-custom-cert.adoc

```diff
@@ -17,6 +17,44 @@ The `route.openshift.io/destination-ca-certificate-secret` annotation can be use
ek1CNFhEVEkwTURnd09URXdNakF4TTFvWApEVEkyTURnd09URXdNakF4TkZvd0tqRW9NQ1lHQTFVRUF3d2ZLaTVoY0hCekxtUndZWFJsY21sNUxYTnVieTVsCmVHRnRjR3hsTG1OdmJUQ0NBU0l3RFFZSktvWklodmNOQVFFQkJRQURnZ0VQQURDQ0FRb0NnZ0VCQUpZeHZSY3kKamM1eFNvcUR4Uk1CMjcrVFFGcmxISWhyaXhvWkJrYXBkV1AwQlJJL3hFTjNiZ3FJUHFDMzZVNTcvUGdQZ3Y0SgpnbWNhOW15K1FDNjg0Rnl1M0gyUU5VNWo3NTZTNE5qNHJ4bHZhbFNaZlRrTnZFQmRiNHJlMVNEWEZnajgrZkNlCnFWU0tTMElFU0EwN3FwL2tMNXJjUUxHVjZnaTJWbWQ2dTRjSFJhZDkvMitCY2JCUEFQSkgwejdjZVJ1TW9aY2MKRzBUUUlsMmhLWHZ1dkxmN3FBNE15R3VRQWEyYThpWW1EQkE5YWNSSXlGcm9rZjZQQk95T2R6WkhFZFJhbnFuNQpSa1gxUFk4SGZMRXdDTmNwQXRaK0d1d0ZiSVpha3g3Z2xiVjM0aHp4OUxXcVVDZXVZQ2NFcVdJY1o5UDIvY2dPCmVWRVlIZ09zSzNmVDZXOENBd0VBQWFPQm9qQ0JuekFPQmdOVkhROEJBZjhFQkFNQ0JhQXdFd1lEVlIwbEJBd3cKQ2dZSUt3WUJCUVVIQXdFd0RBWURWUjBUQVFIL0JBSXdBREFkQmdOVkhRNEVGZ1FVTDR4WEN3dlU0RS8yZ3dxVApWSzB4Rms1azY3NHdId1lEVlIwakJCZ3dGb0FVRzdxQWZOSjRzN0dlVFUrSmdNeXNWWDdvMU1Nd0tnWURWUjBSCkJDTXdJWUlmS2k1aGNIQnpMbVJ3WVhSbGNtbDVMWE51Ynk1bGVHRnRjR3hsTG1OdmJUQU5CZ2txaGtpRzl3MEIKQVFzRkFBT0NBUUVBcjRCSWtwLzBHbExmR2YwTkpLNVBEUDZtWGc4UUpoTHhjajErQjcyQnFmMU0vQUgvekp0bgpnbUUvM1N6VlR1aHFHZk1tdHFKRERqQ29WeU9nWmxjcXpURXhUdVNMeHJIT1FJQWJjVzZTSzVNN1BnL21neEtPCnVZck9ONjE0cjdYL3NOVDRkR2pZOTUzMnlMRi9zTHI4UTdZYWMyYVhiMER4WlEwTW9FS0xxTFZsTEJnc1JWTjEKWEpMU2tZcEFjb3VaaG5EL0EwbWdDbnduSFpDZDlEUzRmZ0V6dzJabEVZbjZ6MjYrcVNIVjgrZ3BYWXVMeVhldgp5YW1ra0M2bjV0Q1EwN3NCWU90YUxVYzJoRmxBUDB1K3l5QnRQbzJkUlR2dUloSGNPaWZNeUt5ZUhuVVlQdzdyCmFvNG95cUhjbHNZblNwOFo1R1hxb1ZUWVZNMmVtOVB0Zmc9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCi0tLS0tQkVHSU4gQ0VSVElGSUNBVEUtLS0tLQpNSUlERERDQ0FmU2dBd0lCQWdJQkFUQU5CZ2txaGtpRzl3MEJBUXNGQURBbU1TUXdJZ1lEVlFRRERCdHBibWR5ClpYTnpMVzl3WlhKaGRHOXlRREUzTWpNeE9UZzRNVE13SGhjTk1qUXdPREE1TVRBeU1ERXlXaGNOTWpZd09EQTUKTVRBeU1ERXpXakFtTVNRd0lnWURWUVFEREJ0cGJtZHlaWE56TFc5d1pYSmhkRzl5UURFM01qTXhPVGc0TVRNdwpnZ0VpTUEwR0NTcUdTSWIzRFFFQkFRVUFBNElCRHdBd2dnRUtBb0lCQVFEUGRrbitacXQ1N2lBbG1rNWg3OEZzCjNodDR5WktHUHh2UlJ2cFlmMG5ndlFuem01U2kyeERqLzlnTGdtU0I0MWNqR2ppaFdEOHhHcmgwY05BSzRQSXoKaGt3eExBZXF2RTZVcTdPMHl5NkVDU0luZHlMMnBRSTZiVkRreitCZnhkQUhxM3hJTk01SC9LeVgvZUZoK0tNZwpBMHZtbTBqYlpET1pCa1hWM2ZUTHpOK01RZDd6S3F2eUZKNGtXVWw5N3hLVkUraElWZDI0bUp2cnBUWHY1UlVRCi9xZkY4K0VzWGFFL0d4WFJmY1hnUVR5Zm5YNjdVZkpOM3BkY1JyM0RXL295ei9xYmRFZjBSMFZQQi9HTjZWWFkKd0RYTDVBTU5WOUYwRytiZGJLZmJPZjBteHJOUXIxc1IxTU56VFpva1pWaW9vRCsvRzAzRFY2WkFwZ2lqaWFpZApBZ01CQUFHalJUQkRNQTRHQTFVZER3RUIvd1FFQXdJQ3BEQVNCZ05WSFJNQkFmOEVDREFHQVFIL0FnRUFNQjBHCkExVWREZ1FXQkJRYnVvQjgwbml6c1o1TlQ0bUF6S3hWZnVqVXd6QU5CZ2txaGtpRzl3MEJBUXNGQUFPQ0FRRUEKRzcyUTcxWGdldlBVWVpLeSt2bmJDMk5mYktMV1E5MXduVFhERkdMVk05Y3JXTi9TRVd3TjFXTHU2TFNHUVEvWApkZmN1cHNRV3hQTktBd3FCVGltUnFWV1hURkxWNjlCSWJHSW1lWW0xOE15blRuTlJ5UzFBaDhFaHBqVlYwNDl1CmlrS21CdVdIR3NLZVFiN3dkYWdBbFBJTFBnNmVJTnJNUzNGWUZuNk9SallaeDBUeXdiTnlacXQzZ25PMCt6U1QKdGR2UTFhVVQ0RUNNTTd1TFF1K1FzLzgwOHdKQUlBc24vK1dpcFBkSVlLcWpFR0NUWnJSOEtDSm9BV3E2Q1djSApnSTBFRXZ4cXhqNC9NN1pSYW1GZ1ZFRnRUeW5FaklFM1c4K3U2cG1XaHZKcWR1T0dtSkF5b0JQNU9GWUo2b3V2Cm83ak54dTAvWGZmdDdINkVOS04xbFE9PQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0tCg==
```

Comment:
You might get a security notification for showing a full example of a secret. I'd replace this with a user-replaced value, such as <server_certificate>. I see this example used for other data.tls.crt entries in our docs.